pkg/graph/schema.graphql

```diff
@@ -76,6 +77,7 @@ type UpdateAccessibilityRequestStatusPayload {
   id: UUID!
   requestID: UUID!
   status: AccessibilityRequestStatus!
+  euaUserId: String!
```

Comment:
This may not be in scope here but...

The payload should just be the AccessibilityRequestStatusRecord and the errors. It shouldn't spread the record's fields here

Reply:
I know it was from a previously approved PR, but I think that we should change the name of this type from `UpdateAccessibilityRequestStatusPayload` to `UpdateAccessibilityRequestStatusRecordPayload`.  We are updating the table, not just the status on that table. `AccessibilityRequestStatus` is an enum.

Reply:
I named it this way because it updates the request's status by adding a new status record. I think `UpdateAccessibilityRequestStatusRecordPayload` would imply that it's updating/editing an individual status record.

Reply:
Aah, yeah, that make sense.  Naming is hard.  I guess with my thinking, it would be `CreateAccessibilityRequestStatusRecordPayload`, which is also kind of confusing.  As is would be fine then- I can't think of a better naming solution.